x/permission/abci.go

```diff
@@ -0,0 +1,12 @@
+package permission
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/bnb-chain/greenfield/x/permission/keeper"
+)
+
+func EndBlocker(ctx sdk.Context, k keeper.Keeper) {
+	k.RemoveExpiredMembers(ctx)
```

Comment:
We may need a limitation to avoid too many db operation in one block

Reply:
Done